jdbc/src/main/java/org/apache/zeppelin/jdbc/JDBCInterpreter.java

```diff
@@ -371,15 +371,16 @@ public void cancel(InterpreterContext context) {
   }
 
   public String getPropertyKey(String cmd) {
-    int firstLineIndex = cmd.indexOf("\n");
-    if (-1 == firstLineIndex) {
-      firstLineIndex = cmd.length();
-    }
-    int configStartIndex = cmd.indexOf("(");
-    int configLastIndex = cmd.indexOf(")");
-    if (configStartIndex != -1 && configLastIndex != -1
-        && configLastIndex < firstLineIndex && configLastIndex < firstLineIndex) {
-      return cmd.substring(configStartIndex + 1, configLastIndex);
+    boolean firstLineIndex = cmd.startsWith("(");
```

Comment:
I think it is not necessary because when I write this query the interpreter what comes is not the name of the interpreter...eg
```
%jdbc (redshift) select * from...
```
to string is applied trim in paragraph class...and the interpreter receives this way without spaces
```
(redshift) select * from...
```
and finally on line 353 is applied trim to the query....what do you think?